edgelet/aziot-edged/src/image_gc.rs

```diff
@@ -155,10 +143,14 @@ async fn garbage_collector(
             )
         })?;
 
-    // delete images
-    for key in image_map.keys() {
-        if let Err(e) = ModuleRegistry::remove(runtime, key).await {
-            log::error!("Could not delete image {} : {}", key, e);
+    if bootstrap_image_id_option.is_some()
```

Comment:
I am aware that this if condition needs to be moved to incorporate the call to prune_images_from_file() on line 136.

That fix will be made with the next push.